02-peer-protocol.md

```diff
@@ -361,28 +355,27 @@ For the `minimum fee` calculation see [BOLT #3](03-transactions.md#calculating-f
 The maximum inputs and outputs are capped at 252. This effectively fixes
 the byte size of the input and output counts on the transaction to one (1).
 
-
 ### The `tx_signatures` Message
 
 1. type: 71 (`tx_signatures`)
 2. data:
     * [`channel_id`:`channel_id`]
     * [`sha256`:`txid`]
-    * [`u16`:`num_witnesses`]
+    * [`bigsize`:`num_witnesses`]
     * [`num_witnesses*witness_stack`:`witness_stack`]
 
 1. subtype: `witness_stack`
 2. data:
-    * [`u16`:`num_input_witness`]
+    * [`bigsize`:`num_input_witness`]
```

Comment:
Can you have more than 65k witnesses? I am incredulous.

Reply:
Same as https://github.com/niftynei/lightning-rfc/pull/3#discussion_r856006049, it's more about being consistent and start using `bigsize` everywhere in the spec for field lenghts. But if you don't like it I can revert that and put `u16` everywhere, it's your call!